docs/K8SCollectors.md

```diff
@@ -0,0 +1,179 @@
+# **K8S data collectors**
+The Splunk Operator for K8S deploys Splunk Enterprise custom resources across a single/multiple namespace/s. The helper scripts `k8s-splunk-collector.sh` and `k8s-systeminfo-collector.sh` in the [tools directory](https://github.com/splunk/splunk-operator/tree/master/tools/k8s_collectors), collect data from a K8S cluster which runs the Splunk Operator for K8S.
+
+## **Splunk Collector for K8S - k8s-splunk-collector.sh**
+The Splunk collector for K8S collects data from multiple <kubectl> get/describe commands, container logs and diags from the Splunk instances(if opted for) in the context of the current namespace.
```

Comment:
Minor: missing space between "instances" and "(if opted for)". Every time in this .md file that there is a parenthetical statement in the text, it appears to have a missing space before the parenthesis. Is this just a formatting glitch?